packages/xrpl/src/models/ledger/Bridge.ts

```diff
@@ -0,0 +1,84 @@
+import { Amount, XChainBridge } from '../common'
+
+import BaseLedgerEntry from './BaseLedgerEntry'
+
+/**
+ * A Bridge objects represents a cross-chain bridge and includes information about
+ * the door accounts, assets, signature rewards, and the minimum account create
+ * amount.
+ *
+ * @category Ledger Entries
+ */
+export default interface Bridge extends BaseLedgerEntry {
+  LedgerEntryType: 'Bridge'
+
+  /** The door account that owns the bridge. */
+  Account: string
+
+  /**
+   * The total amount, in XRP, to be rewarded for providing a signature for
+   * cross-chain transfer or for signing for the cross-chain reward. This amount
+   * will be split among the signers.
+   */
+  SignatureReward: Amount
+
+  /**
+   * The minimum amount, in XRP, required for an {@link XChainAccountCreateCommit}
+   * transaction. If this isn't present, the {@link XChainAccountCreateCommit}
+   * transaction will fail. This field can only be present on XRP-XRP bridges.
+   */
+  MinAccountCreateAmount?: string
```

Comment:
Should optional fields be near the bottom?

Reply:
Does it matter?

Reply:
I think having them at the end makes it easier to read what fields you should be adding when the IDE auto completes. (You don't have to skip over any values). 

But it's a small thing.